### ISSUE TITLE
IOS.rb change for use in balance modules (catalyst 6500)

### ISSUE THREAD

**Issue by bonigeek** [ytti/oxidized#869]
Change the regex the prompt

```
#  prompt /^([\w.@()-]+[#>]\s?)$/
  prompt /^(\r?[\w.@:\/-]+[#>]\s?)$/
```

**Comment by ytti** [comment]
You can pass prompt even in node details. But this seems benign change. Please send PR, and add in commit message ```fixes #869```

**Comment by wk** [comment]
This appears to be related to an abandoned PR. Please re-open a PR for this based on current master if it is still relevant.
